js/cryptocom.js

```diff
@@ -382,8 +386,8 @@ module.exports = class cryptocom extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
-                    'amount': this.parseNumber (this.parsePrecision (this.safeString (market, 'quantity_decimals'))),
-                    'price': this.parseNumber (this.parsePrecision (priceDecimals)),
+                    'amount': this.safeNumber (market, 'quantity_tick_size'),
```

Comment:
@ndubel looks like we have different formats depending on the market:
![image](https://user-images.githubusercontent.com/43336371/203772348-ea2a2a5d-6b31-4b11-a811-eb2480a4fc5a.png)
![image](https://user-images.githubusercontent.com/43336371/203772533-301964e3-c893-4147-b280-98b342057fca.png)

Reply:
why different? https://crypto.com/exchange/trade/HERO_USD all amounts multiple of 100.
https://crypto.com/exchange/trade/LSK_USD - all amounts multiple to 0.1

Reply:
@ndubel When the exchange uses the `TICK_SIZE` precision mode, the precision should be represented as a fraction with the minimum acceptable value. 

So instead of 5 we must set 0.00001
Instead of 10 we have to use 0.0000000001
etc
etc

Apparently the exchange is sending different formats (num of decimals or fractions) depending on the market and that is what is causing this issue I think

Reply:
as I see for both this pairs (and all others) exchange give minimum fraction as you say. For HERO/USD minimum amount fraction is `100`, for LSK/USD is `0.1`. I can't understand where you see different formats?

Reply:
@ndubel 🤦 you're right sorry, I did not notice the price was so small that required such a huge tick size, going to merge it soon!